srtp/srtp.c

```diff
@@ -3820,6 +4162,33 @@ srtp_profile_get_master_salt_length(srtp_profile_t profile) {
   }
 }
 
+srtp_err_status_t
+srtp_get_protect_data_added_length(const srtp_policy_t *policy,
+                                   uint32_t use_mki,
+                                   uint32_t mki_index,
```

Comment:
Here it is `uint32_t` :) Other places it is `unsigned int`. I think the C99(ish) types are generally preferred.